int16 ch1_ref
int16 ch2_ref
bool kill_ref
float32 KP
float32 KI
float32 KD